### PR TITLE
[GAL-3984] [GAL-3985] Admire Token

### PR DESCRIPTION
**Pull request by Rohan-cp**
### Summary of Changes

admire and unadmire tokens
TODO: trigger notification, and handle them correctly

### Web

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-12 at 2 20 42 AM | Screenshot 2023-10-12 at 2 20 27 AM |

| Before                                     | After                                      |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-12 at 2 21 11 AM| Screenshot 2023-10-12 at 2 21 21 AM |

### Mobile
| After                                     | After 2                                 |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-10-12 at 2 19 08 AM | Screenshot 2023-10-12 at 2 19 01 AM |

### Notifications
| Mobile                                     | Web                                |
| ------------------------------------------ | ------------------------------------------ |
| Screenshot 2023-11-15 at 3 21 16 PM | Screenshot 2023-11-15 at 3 33 36 PM |

### Edge Cases
List any common edge cases that you have considered and tested.

### Testing Steps
Can test locally

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

File: apps/mobile/src/components/Feed/AdmireBottomSheet/AdmireBottomSheet.tsx [hunk 1]
```diff
@@ -228,3 +234,77 @@ export function ConnectedPostAdmireList({ feedId }: { feedId: string }) {
     />
   );
 }
+
+export function ConnectedTokenAdmireList({ tokenId }: { tokenId: string }) {
```
[the remark this answers is not in the document]
Comment (Rohan-cp) [reply apps/mobile/src/components/Feed/AdmireBottomSheet/AdmireBottomSheet.tsx]: ohh right, I made a separate file for it and it makes more sense I think

File: apps/web/src/hooks/api/posts/useAdmireToken.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+import { ConnectionHandler, graphql } from 'react-relay';
+import { SelectorStoreUpdater } from 'relay-runtime';
+
+import { useToastActions } from '~/contexts/toast/ToastContext';
+import { useAdmireTokenMutation } from '~/generated/useAdmireTokenMutation.graphql';
+import { AdditionalContext, useReportError } from '~/shared/contexts/ErrorReportingContext';
+import { usePromisifiedMutation } from '~/shared/relay/usePromisifiedMutation';
+import { OptimisticUserInfo } from '~/utils/useOptimisticUserInfo';
+
+export default function useAdmireToken() {
```
Comment (Robinnnnn) [inline apps/web/src/hooks/api/posts/useAdmireToken.ts:11]: question: noticed on web we have `useAdmireToken` and `useRemoveTokenAdmire`. on mobile we have `useToggleTokenAdmire`, both with a lot of shared logic

does it make sense to consolidate everything under the same hook in `/shared`?

**Review by Robinnnnn — COMMENTED** [review]
looks like you have a couple merge conflicts!

**Review by Robinnnnn — COMMENTED** [review]
1. let's make sure to add a transparent border when the button doesn't have a border to avoid shifting contents around
image

2. when hovering over the button, the border turns black and then blue. not sure what the expected hover behavior will be for this button, let's ask fraser

3. let's make sure the admire icon container is a circle

image

4. when a button is actively admired, hovering over it produces an un-admire animation, let's make sure we get signoff on that

File: apps/web/src/scenes/NftDetailPage/NftDetailText.tsx [hunk 3]
```diff
@@ -261,23 +415,22 @@ const StyledOwnerAndCreator = styled(HStack)`
 
 const StyledDetailLabel = styled.div<{ horizontalLayout: boolean; navbarHeight: number }>`
   display: block;
-  max-width: 296px;
-  min-width: 296px;
+  max-width: 420px;
+  min-width: 420px;
```
Comment (kaitoo1) [inline apps/web/src/scenes/NftDetailPage/NftDetailText.tsx:419]: looks like this change might affect mobile resposiveness, maybe can use a % if it's a smaller breakpoint?

prod
CleanShot 2023-11-21 at 18 43 15

dev preview
CleanShot 2023-11-21 at 18 43 22

**Pull request merged**